modules/preview-react/text/lib/Text.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+// type changes no need
+import {type, CanvasTypeHierarchy, CanvasTypeVariants} from '@workday/canvas-kit-react/tokens';
+import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+
+export interface TextProps extends BoxProps {
+  /**
+   * Type token as string with level and size separated by dot.
+   */
+  asToken?: `${keyof CanvasTypeHierarchy}.${'large'|'medium'|'small'}`;
```

Comment:
thoughts on renaming this `tokenLevel` ? i feel like the `as` prop and the `asToken` can get confusing.

```
<StyledLabel
      ref={ref}
      as={Element}
      tokenLevel="subtext.large"
      color={disabled ? 'licorice100' : undefined}
      variant={!disabled ? variant : undefined}
      {...elemProps}
    />
```

Reply:
Agree. Let's make it less confusing 👍